runtime/src/plugin.rs

```diff
@@ -359,15 +361,21 @@ impl Plugin {
         // Initialize Haskell runtime if `hs_init` and `hs_exit` are present,
```

Comment:
nit: but we mention hs_exit here in the comment still though, remove the use of that function.

Reply:
Thanks1